awx/main/isolated/manager.py

```diff
@@ -44,7 +44,7 @@ def __init__(self, cancelled_callback=None, check_callback=None):
     def build_runner_params(self, hosts, verbosity=1):
         env = dict(os.environ.items())
         env['ANSIBLE_RETRY_FILES_ENABLED'] = 'False'
-        env['ANSIBLE_HOST_KEY_CHECKING'] = 'False'
+        env['ANSIBLE_HOST_KEY_CHECKING'] = 'True' if settings.AWX_ISOLATED_HOST_KEY_CHECKING else 'False'
```

Comment:
should be the same as `str(settings.AWX_ISOLATED_HOST_KEY_CHECKING)`, which is how we do it in some other places.